install_config/aggregate_logging.adoc

```diff
@@ -440,12 +440,62 @@ endif::openshift-enterprise[]
 [[scaling-elasticsearch]]
 *Changing the Scale of Elasticsearch*
 
-To scale Elasticsearch up, create more deployments. The deployer provides
-a template to create Elasticsearch deployments:
+If you need to scale up the number of Elasticsearch instances your cluster uses
+it is not as simple as changing the number of replicas. This is due to the nature
+of Persistent volumes and how Elasticsearch is configured to store its data and
+recover the cluster.
+
+Instead, you will need to create a Deployment Configuration for each replica.
+
+The Deployer created templates during installation with the Elasticsearch
+configurations provided to it: `logging-es-template` and `logging-es-ops-template`
+if the Deployer was run with `ENABLE_OPS_CLUSTER=true`.
+
+The node quorum and recovery settings were initially set based on the `CLUSTER_SIZE`
+value provided to the deployer.  Since we are changing the cluster size, those
+values will need to be updated.
+
+Prior to changing the number of replicas, to preserve log data the EFK stack should
+first be scaled down as described in steps 1-4
+link:../upgrading/manual_upgrades.html#manual-upgrading-efk-logging-stack[here].
+
+For the changes in the cluster template and the cluster Deployment Configs we will
+use the following values:
+
+====
+`NODE_QUORUM` is the intended cluster size / 2 (rounded down) + 1.
+So for an intended cluster size of 5 the quorum would be 3.
+
+`RECOVER_EXPECTED_NODES` is the same as the intended cluster size.
+
+`RECOVER_AFTER_NODES` is the intended cluster size - 1.
+====
+
+Edit the template of the cluster you will be scaling up and update the three parameters
+as specified above.
+====
+----
+$ oc edit template logging-es[-ops]-template
+----
+====
+
+In addition to updating the template, all the Deployment Configs for that cluster
+also need to have the three environment variable values above updated.  To list
+the names of the configs for the cluster you can use the following.
+
+====
+----
+$ oc get dc -l component=es[-ops] -o name
```

Comment:
I think that needs to be something using `xargs` then.  `oc edit` doesn't accept label selectors it looks like.